aneris/utils.py

```diff
@@ -625,3 +628,19 @@ def _convert_units(self, df, tostd=True):
             assert((df.units == 'kt').all())
             df.loc[where, numcols(df)] /= 1e3
             df.loc[where, 'units'] = 'Mt'
+
+
+def isin(df=None, **filters):
```

Comment:
very fancy. love it!

Reply:
Yes, if there now was also an elegant way to extend it to pattern based matching i'd propose it directly to pandas.